commit b23316aabffa835ecc516cb81daeef5b9155e8a5
Author: Yuntao Wang <ytcoode@gmail.com>
Date:   Thu May 19 23:06:10 2022 +0800

    selftests/bpf: Add missing trampoline program type to trampoline_count test
    
    Currently the trampoline_count test doesn't include any fmod_ret bpf
    programs, fix it to make the test cover all possible trampoline program
    types.
    
    Since fmod_ret bpf programs can't be attached to __set_task_comm function,
    as it's neither whitelisted for error injection nor a security hook, change
    it to bpf_modify_return_test.
    
    This patch also does some other cleanups such as removing duplicate code,
    dropping inconsistent comments, etc.
    
    Signed-off-by: Yuntao Wang <ytcoode@gmail.com>
    Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
    Acked-by: Yonghong Song <yhs@fb.com>
    Link: https://lore.kernel.org/bpf/20220519150610.601313-1-ytcoode@gmail.com

diff --git a/include/linux/bpf.h b/include/linux/bpf.h
index a3ef078401cf..cc4d5e394031 100644
--- a/include/linux/bpf.h
+++ b/include/linux/bpf.h
@@ -724,7 +724,7 @@ struct btf_func_model {
 #define BPF_TRAMP_F_RET_FENTRY_RET	BIT(4)
 
 /* Each call __bpf_prog_enter + call bpf_func + call __bpf_prog_exit is ~50
- * bytes on x86.  Pick a number to fit into BPF_IMAGE_SIZE / 2
+ * bytes on x86.
  */
 #define BPF_MAX_TRAMP_LINKS 38